src/components/Admin/Approval/Approval.js

```diff
@@ -0,0 +1,150 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import ItemSelector from '../ItemSelector';
+import DataController from '../utils/DataController';
+
+import './Approval.css';
+
+class ImageApproval extends Component {
+  state = {
+    loading: true,
+    images: [],
+    storeList: [{name: 'unknown', id: 'unknown'}]
+  };
+
+  getImages = async () => {
+    return this.data
+      .getImages(false, 10)
+      .then(images => this.setState({images}));
+  };
+
+  lastImageTimestamp = () => {
+    return this.state.images
+      .map(image => image.timestamp)
+      .sort()
+      .pop();
+  };
+
+  nextImage = index => {
+    return this.data
+      .getImages(false, 1, this.lastImageTimestamp())
+      .then(images => {
+        this.setState(prevState => {
+          if (images.length === 1) {
+            prevState.images[index] = images[0];
+          } else {
```

Comment:
Can you make it more explicit that 'else' is triggered when there is no images given? Might read as it will trigger if 2 images are returned.